lib/new.js

```diff
@@ -0,0 +1,6 @@
+var path = require('path'),
```

Comment:
This file also appears to only exist in order to patch over `module._load` instead of injecting dependencies directly. Please inline this and inject dependencies in a manner consistent with the rest of the project.

Reply:
I try to avoid duplication. Anyway I changed